.github/workflows/cd.yml

```diff
@@ -46,13 +46,29 @@ jobs:
         run: |
           aws ecs describe-task-definition --task-definition ${{ vars.ECS_TASK_DEFINITION }} --query taskDefinition > ${{ vars.ECS_TASK_DEFINITION_PATH }}
 
+      - name: Download Parameter Store Values
+        id: ssm-download
+        run: |
+          PARAMETERS_JSON=$(aws ssm get-parameters --query "Parameters[*].{Name:Name,ARN:ARN}" --output json | jq -c '.')
```

Comment:
I think we need to create a name convention here so we only pull the secrets that belong to the Rails app (assuming the same AWS account can store secrets for other apps like the FE app)

Reply:
Sure, this is one of the posible changes that we need to include, another is update the `format_secrets.rb` in case that we need to transform the variable name that we receive in the response